src/components/Storage/Card/Table/Header/ActionHeader/ActionHeader.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Button } from '@rmwc/button';
+import { IconButton } from '@rmwc/icon-button';
+import { Theme } from '@rmwc/theme';
+import { Typography } from '@rmwc/typography';
+import React from 'react';
+
+import { CardActionBar } from '../../../../../common/CardActionBar';
+import { useStorageFiles } from '../../../../api/useStorageFiles';
+import { UseMultiselectResult } from '../../../../common/useMultiselect';
+import { StorageItem } from '../../../../types';
+import styles from './ActionHeader.module.scss';
+
+interface ActionHeaderProps {
+  selection: UseMultiselectResult;
+}
+
+function useAreFoldersSelected(files: StorageItem[], selection: Set<string>) {
```

Comment:
nit: this is not a hook and please remove the `use` prefix to reduce confusion. (It's a stateless pure function whose return value only varies based on input arguments)

Reply:
Ah, good call. done